demo/index.js

```diff
@@ -1,7 +1,7 @@
 import React, { Component } from 'react';
 import ReactDOM from 'react-dom';
 import querystring from 'querystring';
-import ReactSwipe from '../src/ReactSwipe';
+import ReactSwipe from '../src/reactSwipe';
```

Comment:
@voronianski because actual name in filesystem is `src/reactSwipe.js` (with a lowecase 'r').
As I mentioned above it is not an issue for Windows but a big deal for *nix operation systems where file paths are case sensitive.
This is not breaking build process on Windows if you're concerned